content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -119,14 +119,10 @@ process-item-banana-wrsf7   0/1       Completed   0          4m
 process-item-cherry-dnfu9   0/1       Completed   0          4m
 ```
 
-There is not a single command to check on the output of all jobs at once,
-but looping over all the pods is pretty easy:
+We can use this single command to check on the output of all jobs at once:
```

Comment:
```suggestion
For this simple example, we can view the output via logs from the pods:
```
(to emphasize that a different approach is appropriate to read output from production jobs, especially if there are very many of them)

Reply:
@sftim I prefer the original wording. Thank you very much.